orangewidget/gui.py

```diff
@@ -2009,8 +2011,6 @@ def do_commit():
         finally:
             QApplication.restoreOverrideCursor()
 
-    set_dirty(False)
```

Comment:
This is not needed because `toggle_checkbox` calls it.

If decide to keep it, it must be moved to the same place where I moved `checkbox_toggled`.